### DISCUSSION TITLE
Where can i get react project experiences?

### DISCUSSION THREAD

**Discussion by thihaaung225278** [community/community#87952] [GitHub Education]
### Hi everyone,

Now, I am working at developing company with HTML, CSS, vanilla javascript for frontend. I already have some experience for those languages. I want to go next level like react, next. But , where can i get project experience for react? I can find tutorial but I can't spot for the project exprience. If you have some task for react , plz can you share for this task for experience. thanks....

**Comment by MoinulIslam7** [comment]
Hello @thihaaung225278 
To gain more project ideas and find additional resources, you can check out the link [React-Projects](https://hackr.io/blog/react-projects). This platform offers a collection of project ideas along with tutorials and resources to guide you through the implementation process.

Keep in mind that finishing projects and comprehending the ideas and difficulties you face along the road are equally important for acquiring meaningful experience. I wish you well on your React adventure!

**Reply by thihaaung225278** [reply]
Thanks, bro @MoinulIslam7, and I appreciate your sharing.

**Comment by Kavin-Brat** [comment]
Hey @thihaaung225278,

Great to see your interest in gaining experience with React! There are plenty of courses available, and the React documentation is now very clear. To get started easily, try working on small React projects like the one @MoinulIslam7 shared. 

Showcase your projects, and as you become more proficient, consider taking on projects within your company or elsewhere. It's all about doing and learning!

Best of luck! 😊

**Comment by shanakasp** [comment]
HI, as a training SE I will express my thoughts,  I think that is better first of all identify what is the main use of react. Then you may get to know about react components, hooks, props, . then create a mind map containing the major parts of react. then try to learn from projects. It would better you can follow courses too. but in my case of course it is too boring. try to watch youtube react projects with some suitable backend and db approach, my recommendation is MERN project. Good luck
